### PR TITLE
Unet merge

### PR DESCRIPTION
**Pull request by Kevin2**
merging pointsource tests

### CODE REVIEW

File: mnist_cnn/Makefile [hunk 1]
```diff
@@ -60,7 +60,6 @@ find_lr: ./lr_find
 		${arch}\
 
 ./lr_find:
-		mkdir -p models/
```
Comment (FritzGeise) [inline mnist_cnn/Makefile:63]: Why delete the mkdir?

File: mnist_cnn/training [hunk 2]
```diff
@@ -0,0 +1,97 @@
+#########################################################
```
Comment (FritzGeise) [inline mnist_cnn/training:1]: Is this a Makefile?

File: mnist_cnn/visualize/predictions.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import click
```
Comment (FritzGeise) [inline mnist_cnn/visualize/predictions.py:1]: Why create the visualize folder again? I thought we refactored it

**Comment by Kevin2** [comment]
I think the PR will not be needed anymore after the restructuring is completed.

**Pull request closed without merging**